docs/Gopkg.toml.md

```diff
@@ -221,6 +221,36 @@ Almost all projects will be fine without setting any project-specific rules, and
 ```
 It is usually safe to set `non-go = true`, as well. However, as dep only has a clear model for the role played by Go files, and non-Go files necessarily fall outside that model, there can be no comparable general definition of safety.
 
+## Notes
+
+`dep` evaluates
+
+* `[[override]]`
+* `[[ignore]]`
+* `require`
+* `ignored`
+
+only in the root project, _ie_ the project where `dep` runs.For example,
+
+You have a project: `github.com/yourusername/awesomeproject`. 
+
+`github.com/foo/bar` is dependecy for your project.
+
+Here `dep` evaluates the `Gopkg.toml` files of both these packages as follows.
+
+```
+github.com/urname/goproject     |      github.com/foo/bar
```

Comment:
For some reason this is rendering wrong on github. Any ideas?

https://github.com/scriptonist/dep/blob/06a66e4f595c87d8d5c99ddf60b5f6ac30a14cf2/docs/Gopkg.toml.md